pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -265109,7 +265109,6 @@ self: {
        description = "Multi-backend, high-level EDSL for interacting with SQL databases";
        license = lib.licenses.mit;
        hydraPlatforms = lib.platforms.none;
```

Comment:
@ncfavier You have to remove the `hydraPlatforms = none` line as well, otherwise Hydra won't build the package.

Reply:
Oops. So I guess you have to run `regenerate-hackage-packages.sh`, not just `regenerate-transitive-broken.sh`

Reply:
Ah, I see, I had thought you had just done this by hand!